packages/docs/src/routes/docs/(qwikcity)/action/index.mdx

```diff
@@ -58,6 +58,7 @@ export default component$(() => {
 
 > Since actions are not executed during rendering, they can have side effects such as writing to a database, or sending an email. An action only runs when called explicitly.
 
+> Actions must be declared on top of the route files, so we must declare it either on `layout.tsx` or `index.tsx` otherwise Qwik will not execute them.
```

Comment:
do you think this is redundant with these info?
https://qwik.builder.io/docs/action/#routeaction-1

Reply:
Hi @zanettin, yes I think so. The only suggestion I can make is highlight it in somehow. Let's disregard this PR. 👍